intro-to-tests/intro-to-pytest.md

```diff
@@ -192,6 +192,18 @@ def test_zero_division():
 | `ZeroDivisionError`       | The kind of exception we expect should be specified here                                      |
 | `1 / 0`                   | In this code block, we should do our **Act** step, or invoke the function we're testing here. |
 
+### !callout-danger
+
+## This Curriculum Uses unittest, Not pytest
+This curriculum teaches pytest, a testing framework that has wide adoption in the Python community. This curriculum's projects will use pytest. However, **the code challenges within this curriculum will use unittest,** a different testing framework. This is due to a limitation in the curriculum tools.
+
+---
+
+Working with unittest, rather than pytest, will have some differences in syntax. However, conceptually, they are the exact same; the structure, error output, error messages, passed test messages, etc. will all be extremely alike.
+
+This curriculum believes in a learner's ability to smoothly work with these differences.
```

Comment:
Perhaps add one more line, "In industry, teams are often evolving to use the latest and greatest tools available. As a result, it's common for developers to learn new frameworks on the job."